File: gxylib/src/main/java/com/srwing/gxylib/recycle_adapter/adapter/delegate/AdapterDelegate.java
package com.srwing.gxylib.recycle_adapter.adapter.delegate;

import java.util.List;

/**
 * Description:
 * Created srwing
 * Date: 2022/9/28
 * Email: 694177407@qq.com
 */
public interface AdapterDelegate<T> {
    void setList(List<T> datas);

    void addList(List<T> datas);
}

File: gxylib/src/main/java/com/srwing/gxylib/recycle_adapter/adapter/delegate/AdapterDelegate.kt
package com.srwing.gxylib.recycle_adapter.adapter.delegate

/**
 * Description:
 * Created by srwing
 * Date: 2022/7/7
 * Email: 694177407@qq.com
 */
interface AdapterDelegate<T> {
    fun setList(datas: List<T>?)
    fun addList(datas: List<T>?)
}